### PR TITLE
Fix some bugs

### PR DESCRIPTION
**Pull request by mfilippov**
- Fix usage bundled Java on macOS
- Add accept EULA in L2Tests
- Fix UI freeze on project loading

### CODE REVIEW

**Comment by mfilippov** [comment]
@kaylangan Hi, PR is ready to review.

File: plugin/src/com/microsoft/alm/plugin/external/utils/ProcessHelper.java [hunk 1]
```diff
@@ -27,6 +27,6 @@ public static Process startProcess(final String workingDirectory, final List<Str
     private static String getPatchedPathWithCurrentJavaBinLocation() {
         String currentJavaHome = System.getProperty("java.home");
         String originPathVariable = System.getenv("PATH");
-        return originPathVariable + File.pathSeparator + currentJavaHome + File.separator + "bin" + File.separator;
+        return File.pathSeparator + currentJavaHome + File.separator + "bin" + File.separator + File.pathSeparator + originPathVariable;
```
Comment (jeschu1) [inline plugin/src/com/microsoft/alm/plugin/external/utils/ProcessHelper.java]: Why does this need to start with File.pathSeparator?

Comment (mfilippov) [reply plugin/src/com/microsoft/alm/plugin/external/utils/ProcessHelper.java]: @jeschu1 Fixed.

File: plugin.idea/src/com/microsoft/alm/plugin/idea/common/utils/VcsHelper.java [hunk 2]
```diff
@@ -93,8 +95,13 @@ public static boolean isVstsRepo(final Project project) {
             if (!isGitVcs(project)) {
                 return false;
             }
-            final GitRepository repo = GitUtil.getRepositoryManager(project).getRepositoryForFile(project.getBaseDir());
-            return repo != null && TfGitHelper.isTfGitRepository(repo);
+
+            return ContainerUtil.exists(GitUtil.getRepositoryManager(project).getRepositories(), new Condition<GitRepository>() {
+                @Override
+                public boolean value(GitRepository gitRepository) {
+                    return TfGitHelper.isTfGitRepository(gitRepository);
```
Comment (jeschu1) [inline plugin.idea/src/com/microsoft/alm/plugin/idea/common/utils/VcsHelper.java]: Do we need to check for null, like we previously did?

Comment (mfilippov) [reply plugin.idea/src/com/microsoft/alm/plugin/idea/common/utils/VcsHelper.java]: @jeschu1  getRepositories() marked as NotNull and I think returned list shouldn't contain null elements. In this case, the null check not needed.

**Comment by mfilippov** [comment]
@kaylangan Could you merge this PR or need approve from the second reviewer?

**Comment by kaylangan** [comment]
@mfilippov merged. Don't have an ETA on a new release yet.

**Comment by mfilippov** [comment]
@kaylangan Thank you.

**Pull request merged**